### PR TITLE
reduce maintenance by using git as source

### PR DESCRIPTION
**Pull request by shimwell**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR aims to reduce the maintenance of keeping this recipe up to date by making use of a git source instead of source urls. Making use of git removes then need to specify and keep up to date sha values and also save a few lines. The submodules of the git repo are brought in automatically with the versions specified in the git repo so if they change there would b no need to update this metal.yml and not need to be specified in two places.

### CODE REVIEW

**Comment by shimwell** [comment]
This has been tested locally with the command ```python build-locally.py  linux_64_numpy1.19python3.9.____cpython``` as suggested [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#running-tests-locally-for-staged-recipes) by conda-forge

**Comment by shimwell** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,17 @@
...
-    folder: vendor/xtensor
-  - url: https://github.com/xtensor-stack/xtl/archive/0.6.13.tar.gz
-    sha256: 12bd628dee4a7a55db79bff5a47fb7d96b1a4dba780fbd5c81c3a15becfdd99c
-    folder: vendor/xtl
-  - url: https://github.com/fmtlib/fmt/archive/d141cdbeb0fb422a3fb7173b285fd38e0d1772dc.tar.gz
-    sha256: 8d2cb0daccef71cbf3ee8068a845fdd379545acd289873e67c41fdc1b530bfcd
-    folder: vendor/fmt
+  git_url: https://github.com/openmc-dev/{{ name }}.git
+  git_rev: v{{ version }}
+  git_depth: 1 # (Defaults to -1/not shallow)
```
Comment (kkiesling) [inline recipe/meta.yaml:11]: It looks like the build logs have been removed so I can't tell for sure, but try removing this line - I think it is not getting commits far enough back to access the correct tag.

Comment (shimwell) [reply recipe/meta.yaml]: Thanks for the suggestion, I've given this a go with the commit below.

**Comment by shimwell** [comment]
closing this PR as it has got quite out of date with the main line.

Opening a new PR https://github.com/conda-forge/openmc-feedstock/pull/50 instead

**Pull request closed without merging**